src/indexes.rs

```diff
@@ -285,40 +285,49 @@ impl Index {
     /// # let MEILISEARCH_HOST = option_env!("MEILISEARCH_HOST").unwrap_or("http://localhost:7700");
     /// # let MEILISEARCH_API_KEY = option_env!("MEILISEARCH_API_KEY").unwrap_or("masterKey");
     /// #
-    /// #[derive(Serialize, Deserialize, Debug)]
-    /// # #[derive(PartialEq)]
+    /// #[derive(Serialize, Debug)]
     /// struct Movie {
     ///    name: String,
     ///    description: String,
+    ///    age: Option<usize>
+    /// }
+    ///
+    /// #[derive(Deserialize, Debug)]
+    /// # #[derive(PartialEq)]
```

Comment:
```suggestion
    /// #[derive(Deserialize, Debug, PartialEq)]
```

Since you show the `assert_eq` later you should also show that your documents implements PartialEq